databricks-kube/src/context.rs

```diff
@@ -19,28 +19,59 @@ pub struct Context {
     api_secret_store: Arc<Store<Secret>>,
 }
 
-#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema)]
+#[derive(Clone, Deserialize, Serialize, PartialEq, JsonSchema, Debug)]
 pub struct DatabricksAPISecret {
```

Comment:
You can make these `Option<String>` and `#[derive(Default)]` -- the trait will see the options and use their defaults, which are None. Mainly -- the default should have some kind of useful semantics, in this case I think 'None' is better.